scripts/aggregate.py

```diff
@@ -87,7 +88,12 @@ def failures_to_markdown(failures: Dict[str, List[str]], current_hash: str, patc
     with open("./labels.txt", "w") as f:
         f.write(f"{','.join(labels)}")
     result += "---\n\n"
-    result += failures_to_summary(failures)
+    summary, no_failures = failures_to_summary(failures)
+    result += summary
+    if no_failures:
```

Comment:
with `assert(len(build_failures) != 0 or len(testsuite_failures) != 0 or len(failures) != 0)`, the assert will trip if `len(build_failures) != 0 or len(testsuite_failures) != 0 or len(failures) != 0` results in `False`. Therefore, shouldn't it be `if not no_failures`?

Reply:
Whoops thanks for catching this - I moved the not up above so the variable name is accurate.